src/shared/components/mutationTable/column/FunctionalImpactColumnFormatter.tsx

```diff
@@ -60,14 +60,13 @@ class FunctionalImpactColumnTooltip extends React.Component<
     }
 
     legend() {
-        const showMutationAssessor = shouldShowMutationAssessor();
```

Comment:
Is there a specific reason why we want to call the `shouldShowMutationAssessor` function multiple times instead of once?

Reply:
Because sometimes serverConfig is not ready when it runs `const showMutationAssessor = shouldShowMutationAssessor();`, which would be false and doesn't refresh. But good point, better to have a flag const rather than calling the function itself.